Formula/lilypond.rb

```diff
@@ -36,14 +45,17 @@ class Lilypond < Formula
   depends_on "python@3.9"
 
   uses_from_macos "flex" => :build
+  uses_from_macos "perl" => :build
+  uses_from_macos "rsync" => :build
+  uses_from_macos "zip" => :build
```

Comment:
I'm not sure these are needed, since the Linux build succeeded in the original PR. I'm pretty sure we don't have `rsync` on the Linux runner. (Unsure about `zip`.)

Reply:
We definitely don't on self-hosted, maybe on `ubuntu-latest`. Can do a quick run on the self-hosted if necessary.

Reply:
Yeah we definitely don't have `rsync` - I know this from the Swift PR.

Reply:
I’ve removed `rsync` and `zip`.